### PR TITLE
[13.0][MIG] stock: Recompute complete_name in stock locations.

### PR DESCRIPTION
**Pull request by sergio-teruel**
cc @Tecnativa

### CODE REVIEW

File: addons/stock/migrations/13.0.1.1/post-migration.py [hunk 1]
```diff
@@ -319,6 +319,11 @@ def map_stock_locations(env, main_company):
     """)
 
 
+def recompute_stock_location_comple_name(env):
```
Comment (legalsylvain) [inline addons/stock/migrations/13.0.1.1/post-migration.py]: typo : ``recompute_stock_location_comple_name`` --> ``recompute_stock_location_complete_name``

Comment (legalsylvain) [inline addons/stock/migrations/13.0.1.1/post-migration.py:322]: typo. complete_name.

**Review by legalsylvain — COMMENTED** [review]
Hi. thanks for the patch. could you mention why complete_name should be recomputed ? 

thanks !

**Comment by sergio-teruel** [comment]
Hi @legalsylvain Method comment added....
Explanation:
In 12.0, the displayed name of the locations was obtained by the name_get method. This method limited the location path up to a parent location of type view is found. In 13.0, complete_name is used for this purpose, and this one used to be stored with full path in v12. We need to recompute it.

**Review by chienandalu — APPROVED** [review]

**Review by StefanRijnhart — APPROVED** [review]

**Review by kos94ok-3D — APPROVED** [review]

**Pull request merged**